docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -722,6 +722,52 @@ directly or used in conditional statements.
 * ``$file_size``
     Size of the file for this row.
 
+Functions
+---------------
+
+The connector exposes following functions:
+
+table_changes
+^^^^^^^^^^^^^
+
+Allows reading Change Data Feed (CDF) entries. This feature allows
+Delta tables to track row-level changes between versions of a Delta table.
+When enabled on a Delta table, it records "change events" for all the data
+written into the table.
+
+.. code-block:: sql
+
+    SELECT * FROM TABLE(system.table_changes(schema_name => 'test_schema', table_name => 'tableName', since_version => 0))
+
+``since_version`` - type ``BIGINT``, exclusive, changes will be read from ``since_version + 1``
```

Comment:
This is confusing. I don't understand what it means that "changes will be read from since_version + 1". Changes are the delta *between* versions, so they would represent what happened when transitioning from `since_version` to the version following.

Reply:
> so they would represent what happened when transitioning from since_version to the version following

you make an assumption that since_version is exclusive, when in fact in delta lake it is [inclusive](https://docs.databricks.com/delta/delta-change-data-feed.html#read-changes-in-batch-queries), I don't know how to express it better. I wanted to be precise, as you can see it is confusing in one or another way

Reply:
That's not my assumption. My assumption is that versions represent "snapshots" of the database at different points in time, and the deltas are the actions that happen to bring one version to another. E.g.,

* version 0 has a table `t` with no rows
* version 1 has a table `t` with one row

The delta between version 0 and 1 is the "insertion of the row into table t".

Is my understanding incorrect?